RuslanImankulov/homework1.js

```diff
@@ -0,0 +1,61 @@
+//Func to check palindrome
+const isPalindrome = (str) => {
+    let i = 0;
```

Comment:
Well, I think you don't need these additional variables, try to do it only with `for` counter.